recipes/sagemaker-studio-image-build/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sagemaker-studio-image-build" %}
+{% set version = "0.6.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sagemaker_studio_image_build-{{ version }}.tar.gz
+  sha256: b7586597ca2395a63e463f8e3ab981ae987d5ea500bf86096af9a79dc41ec9b5
+
+build:
+  number: 0
+  entry_points:
+    - sm-docker=sagemaker_studio_image_build.cli:main
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [win]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - boto3 >=1.10.44,<2.0
+    - pandas
+    - pathos
```

Comment:
I don't see either of these in any of the code for the tarball for this.

Reply:
thanks for digging in here.

See https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=439070&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1233
Looks as though the build fails without these. Any suggestions?

Reply:
Make a PR to sagemaker-python-sdk? I see pathos missing there in a recent PR by autotick bot https://github.com/conda-forge/sagemaker-python-sdk-feedstock/pull/127#issue-1088670416